internal/commands/pull/command_test.go

```diff
@@ -234,24 +252,70 @@ Successfully pulled app down: app
 			return "app_20210101", &zipPkg.Reader, nil
 		}
 		realmClient.ExportDependenciesFn = func(groupID, appID string) (string, io.ReadCloser, error) {
```

Comment:
is this needed for this test case?  or are we just calling `ExportDependenciesArchive` here?